specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-02-01/management.json

```diff
@@ -0,0 +1,1997 @@
+        },
+        "defaultManagementGroup": {
+          "type": "string",
+          "description": "Settings that sets the default Management Group under which new subscriptions get added in this tenant. For example, /providers/Microsoft.Management/managementGroups/defaultGroup"
+        }
+      }
+    },
+    "HierarchySettingsList": {
+      "description": "Lists all hierarchy settings.",
+      "properties": {
+        "value": {
+          "description": "The list of hierarchy settings.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HierarchySettingsInfo"
+          }
+        },
+        "nextLink": {
+          "description": "The URL to use for getting the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "HierarchySettingsInfo": {
+      "description": "The hierarchy settings resource.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the settings object.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000/settings/default.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource.  For example, Microsoft.Management/managementGroups/settings.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the object. In this case, default.",
+          "readOnly": true
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "title": "Properties",
+          "$ref": "#/definitions/HierarchySettingsProperties"
+        }
+      }
+    },
+    "CreateOrUpdateSettingsRequest": {
+      "description": "Parameters for creating or updating Management Group settings",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "properties": {
+          "title": "Properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CreateOrUpdateSettingsProperties"
+        }
+      }
+    },
+    "CreateOrUpdateSettingsProperties": {
+      "description": "The properties of the request to create or update Management Group settings",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "requireAuthorizationForGroupCreation": {
+          "type": "boolean",
+          "description": "Indicates whether RBAC access is required upon group creation under the root Management Group. If set to true, user will require Microsoft.Management/managementGroups/write action on the root Management Group scope in order to create new Groups directly under the root. This will prevent new users from creating new Management Groups, unless they are given access."
+        },
+        "defaultManagementGroup": {
+          "type": "string",
```

Comment:
this is the definition used in patch.

Are we sure all these properties are patchable ?

Reply:
Yes, they are. It's a simple switch for RBAC checks on our side.